src/components/Card/card.module.scss

```diff
@@ -30,11 +31,17 @@
 
   &-large {
     height: 278px;
+    padding: $space-ml;
+  }
+
+  &-medium {
+    height: 260px;
+    padding: $space-m;
   }
 
-  &-medium,
   &-small {
     height: 260px;
+    padding: $space-s;
```

Comment:
and regarding prescriptive heights, There is so little variance in these, hard to justify calling the property "size" when mostly it just tweaks padding.m My 2cents

Reply:
Needed to play it safe, reducing the height of `small` as it was previously `medium` was a risk.